themes/default/content/docs/languages-sdks/python/python-blocking-async.md

```diff
@@ -0,0 +1,206 @@
+---
+title_tag: "Blocking and Async Python with Pulumi | Languages & SDKs"
+meta_desc: An overview of how to use Python with Pulumi for infrastructure as code on any cloud (AWS, Azure, Google Cloud, Kubernetes, etc.).
+title: Blocking and Async
+h1: Blocking and Async Python with Pulumi
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  languages:
+    parent: python
+    weight: 5
+aliases:
+- /python/async/
+---
+
+A Python Pulumi program runs an
+[`asyncio`](https://docs.python.org/3/library/asyncio.html)) event loop to
+create the resources that the Pulumi program describes---this isn't directly
+visible to the Python program.
+
+A consequence of this is that blocking calls prevent progress from being made by
+the Pulumi runtime by preventing the event loop from processing asynchronous
+tasks.  
+
+The `pulumi.Output` object can be used to pass the output from both synchronous
+and asynchronouse calls without blocking the event loop.
```

Comment:
```suggestion
and asynchronous calls without blocking the event loop.
```

Reply:
aargh, I always make this one typo! It makes me think of a mouse